Gemfile

```diff
@@ -48,28 +48,29 @@ group :development do
   gem "awesome_print"
   gem "defra_ruby_style"
   gem "puma"
+  gem "rubocop"
+  gem "rubocop-rails"
+  gem "rubocop-rspec"
```

Comment:
Not sure why do you need to add "rubocop" and "rubocop-rspec" explicitly if we have it installed via "defra_ruby_style" gem?

Reply:
![image](https://github.com/DEFRA/flood-risk-back-office/assets/4951358/3172bb98-5554-4698-ae77-737dfa4c7e91)